Formula/gst-validate.rb

```diff
@@ -23,6 +20,7 @@ class GstValidate < Formula
 
   depends_on "gobject-introspection" => :build
   depends_on "pkg-config" => :build
+  depends_on "python" => :test
```

Comment:
Why is this needed? Sound like an issue - we don't want anything using 3.7 anymore.